src/CommonDBRelation.php

```diff
@@ -2142,9 +2142,9 @@ public function getForbiddenStandardMassiveAction()
      */
     public static function getMemberPosition(string $class): int
     {
-        if ($class == static::$itemtype_1) {
+        if ($class == static::$itemtype_1 || is_subclass_of($class, static::$itemtype_1)) {
```

Comment:
```suggestion
        if (is_a($class, static::$itemtype_1, true)) {
```

Reply:
I tried this solution but it does not works because $class is a class name, not an instance. is_subclass_of does not includes the tested parent class, then I think we're ought to test with is_a() then is_subclass_of()

Reply:
Are you sure ? The third parameter  of `is_a` set to true specifically allow class names instead of instances.

Reply:
Ah yes, I did not cared about this argument. I'll test it.

Reply:
it works, thanks for the tip @AdrienClairembault